specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2024-03-02/examples/diskExamples/Disk_Update_AddSupportsHibernation.json

```diff
@@ -0,0 +1,60 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2024-03-02",
+    "diskName": "myDisk",
+    "disk": {
+      "properties": {
+        "supportsHibernation": true
+      }
+    }
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Location": "https://management.azure.com/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/disks/myDisk?api-version=2021-04-01"
```

Comment:
>2021-04-01 [](http://example.com/codeflow?start=165&length=10)

Please update  - replace all "2021-04-03"

Reply:
@madhura-np  thank you for the finding I've addressed all the missed version changes in the examples